pkg/manifest/subscription.go

```diff
@@ -177,6 +177,14 @@ func subscriptionService(
 			}
 		}
 	}
+	// Enable content sets if specified
+	if len(subscriptionOptions.ContentSets) > 0 {
+		contentSetsCmd := "/usr/sbin/subscription-manager repos"
+		for _, contentSet := range subscriptionOptions.ContentSets {
+			contentSetsCmd += fmt.Sprintf(" --enable=%s", contentSet)
```

Comment:
Let's be nice here and use `shutil.Quote(contentSet)` from `github.com/osbuild/images/pkg/shutil` :)